js/about/bookmarks.js

```diff
@@ -55,11 +55,19 @@ class BookmarkItem extends ImmutableComponent {
         <span>&nbsp;(<span data-l10n-id='partitionNumber' data-l10n-args={JSON.stringify(l10nArgs)} />)</span>
     }
 
+    var className = 'listItem'
+
+    // If the bookmark item is in the selected folder, show
+    // it as selected
+    if (this.props.inSelectedFolder) {
+      className += ' selected'
+    }
```

Comment:
A better way to do this is with `cx` which allows you to selectively include or exclude css styles with a boolean expression.  You can find some uses in the same file.